common/presenters/population-table/population-table-config.js

```diff
@@ -0,0 +1,73 @@
+const { addDays, format, isSameDay } = require('date-fns')
+
+const locationMetaToSpacesLink = require('./location-meta-to-spaces-link')
+
+const populationTableDay = {
+  establishmentConfig({ isSortable = false }) {
+    return {
+      head: {
+        isSortable,
+        sortKey: 'title',
+        html: 'Establishment',
```

Comment:
Could this use the translations? I know it feels a little overkill but it keeps it consistent with how we do content everywhere else in the app.

Reply:
Updated this to use the translations